src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -61,20 +106,50 @@ const collectorSignalsLoader = async (
 
   // Handle signals for auction artworks
   if (isInSale && auctionsCollectorSignalsEnabled) {
-    const activeSaleArtwork = await getActiveSaleArtwork(
+    const activeLotData = await getActiveSaleArtwork(
       {
         artworkId,
         saleIds: artwork.sale_ids,
       },
       ctx
     )
 
-    if (activeSaleArtwork) {
+    const activeAuctionLot = activeLotData && !activeLotData.sale.ended_at
+    if (activeAuctionLot) {
+      const { saleArtwork, sale } = activeLotData
+
       if (artwork.recent_saves_count) {
         lotWatcherCount = artwork.recent_saves_count
       }
-      if (activeSaleArtwork.bidder_positions_count) {
-        bidCount = activeSaleArtwork.bidder_positions_count
+
+      if (saleArtwork.bidder_positions_count) {
+        bidCount = saleArtwork.bidder_positions_count
+      }
+
+      const futureRegistrationEndTime = ifFutureDate(sale.registration_ends_at)
+      if (futureRegistrationEndTime) {
+        registrationEndsAt = futureRegistrationEndTime
+      }
+
+      const saleLiveStartAt = sale.live_start_at
+      if (saleLiveStartAt) {
+        const futureLiveStartTime = ifFutureDate(saleLiveStartAt)
+
+        if (futureLiveStartTime) {
+          liveBiddingStarted = false
+          liveStartAt = futureLiveStartTime
+        } else {
+          liveBiddingStarted = true
+        }
+      }
+
+      const extendedBiddingEndAt = saleArtwork.extended_bidding_end_at
+      if (extendedBiddingEndAt && extendedBiddingEndAt.length > 0) {
```

Comment:
Question: what does `extendedBiddingEndAt.length > 0` do?

Reply:
Removed, not needed.